modules/account-lib/.gitignore

```diff
@@ -8,3 +8,5 @@ mochawesome-report/
 package/
 .idea/
 coverage.lcov
+modules/account-lib/resources/trx/protobuf/*.d.ts
```

Comment:
Unfortunately this won't actually ignore the intended files, since the paths are evaluated relative to the .gitignore file. Please remove the `modules/account-lib/` prefix to correct.